diff --git a/dist/agent.js b/dist/agent.js
index 3340b2131a6af7872e5ce7baf401a376460dc38c..5e169137dc9af7d1bc9cf44a96dc9d9eba886b79 100644
--- a/dist/agent.js
+++ b/dist/agent.js
@@ -59,6 +59,9 @@ export class Agent extends XrpcClient {
             if (this.proxy && !headers.has('atproto-proxy')) {
                 headers.set('atproto-proxy', this.proxy);
             }
+            if (headers.get('atproto-proxy') === '') {
+                headers.delete('atproto-proxy');
+            }
             // Merge the labelers header of this particular request with the app &
             // instance labelers.
             headers.set('atproto-accept-labelers', [
@@ -399,7 +402,7 @@ export class Agent extends XrpcClient {
                 hideAllFeeds: false,
             },
         };
-        const res = await this.app.bsky.actor.getPreferences({});
+        const res = await this.app.bsky.actor.getPreferences({}, {headers: {'atproto-proxy': ''}});
         const labelPrefs = [];
         for (const pref of res.data.preferences) {
             if (predicate.isValidAdultContentPref(pref)) {
