diff --git a/dist/expo-oauth-client.native.js b/dist/expo-oauth-client.native.js
index 979db40b3398f46a456b249c8335dceb65527341..300a11c428b398ea1c9ba3329a135f7f5f870a9e 100644
--- a/dist/expo-oauth-client.native.js
+++ b/dist/expo-oauth-client.native.js
@@ -62,6 +62,30 @@ const runtimeImplementation = {
     digest: async (bytes, { name }) => NativeModule.digest(bytes, name),
     getRandomValues: async (length) => NativeModule.getRandomValues(length),
 };
+class DisposableStack {
+    stack;
+    constructor() {
+        this.stack = [];
+    }
+    use(value) {
+        return __addDisposableResource({ stack: this.stack, error: void 0, hasError: false }, value, false);
+    }
+    async dispose() {
+        await __disposeResources({ stack: this.stack, error: void 0, hasError: false });
+    }
+    move() {
+        const newStack = new DisposableStack();
+        newStack.stack = this.stack;
+        this.stack = [];
+        return newStack;
+    }
+    [Symbol.dispose]() {
+        this.dispose();
+    }
+    [Symbol.asyncDispose]() {
+        return this.dispose();
+    }
+}
 export class ExpoOAuthClient extends OAuthClient {
     #disposables;
     constructor(options) {
diff --git a/dist/polyfill.native.js b/dist/polyfill.native.js
index 32656aa97e503bb1743f74d0d9214c59ed416213..a2fe17e2c2ec2029edd6da730ad51a379f6f6182 100644
--- a/dist/polyfill.native.js
+++ b/dist/polyfill.native.js
@@ -1,3 +1,8 @@
-import 'core-js/proposals/explicit-resource-management';
+if (typeof Symbol.dispose === 'undefined') {
+  Symbol.dispose = Symbol('Symbol.dispose');
+}
+if (typeof Symbol.asyncDispose === 'undefined') {
+  Symbol.asyncDispose = Symbol('Symbol.asyncDispose');
+}
 import 'react-native-url-polyfill/auto';
 //# sourceMappingURL=polyfill.native.js.map
\ No newline at end of file
diff --git a/dist/polyfill.web.js b/dist/polyfill.web.js
index b7dc3e4fd07707d3fc226a1e8a7f76896e9734d8..8f4a8ef355208465ae9e99e70ee871414ca895a0 100644
--- a/dist/polyfill.web.js
+++ b/dist/polyfill.web.js
@@ -1,2 +1,7 @@
-import 'core-js/proposals/explicit-resource-management';
+if (typeof Symbol.dispose === 'undefined') {
+  Symbol.dispose = Symbol('Symbol.dispose');
+}
+if (typeof Symbol.asyncDispose === 'undefined') {
+  Symbol.asyncDispose = Symbol('Symbol.asyncDispose');
+}
 //# sourceMappingURL=polyfill.web.js.map
