diff --git a/app.config.js b/app.config.js
index e472cc86d..5f6ad0dd8 100644
--- a/app.config.js
+++ b/app.config.js
@@ -24,6 +24,7 @@ module.exports = function (_config) {
 
   const ASSOCIATED_DOMAINS = [
     'applinks:witchsky.app',
+    'applinks:deer.social',
     'applinks:bsky.app',
     // When testing local services, enter an ngrok (et al) domain here. It must use a standard HTTP/HTTPS port.
     ...(IS_DEV || IS_TESTFLIGHT ? [] : []),
diff --git a/src/Navigation.tsx b/src/Navigation.tsx
index 5a6c80c0b..e9db96a1b 100644
--- a/src/Navigation.tsx
+++ b/src/Navigation.tsx
@@ -911,13 +911,15 @@ const FlatNavigator = ({
 
 const LINKING = {
   // TODO figure out what we are going to use
-  // note: `bluesky://` and `witchsky://` is what is used in app.config.js
+  // note: `bluesky://` and `witchsky://` and `deersocial://` is what is used in app.config.js
   prefixes: [
     'bsky://',
     'bluesky://',
     'witchsky://',
+    'deersocial://',
     'https://bsky.app',
     'https://witchsky.app',
+    'https://deer.social',
   ],
 
   getPathFromState(state: State) {
diff --git a/src/lib/strings/url-helpers.ts b/src/lib/strings/url-helpers.ts
index 92e538952..a8f48454e 100644
--- a/src/lib/strings/url-helpers.ts
+++ b/src/lib/strings/url-helpers.ts
@@ -11,6 +11,7 @@ export const BSKY_APP_HOST = 'https://witchsky.app'
 const BSKY_TRUSTED_HOSTS = [
   'witchsky\\.app',
   'witchsky\\.social',
+  'deer\\.social',
   'bsky\\.app',
   'bsky\\.social',
   'blueskyweb\\.xyz',

