diff --git a/src/state/queries/ai-preferences/index.ts b/src/state/queries/ai-preferences/index.ts
index d8106b14a..b8f408d36 100644
--- a/src/state/queries/ai-preferences/index.ts
+++ b/src/state/queries/ai-preferences/index.ts
@@ -48,6 +48,10 @@ export function useAIPreferencesQuery() {
           repo: currentAccount.did,
           collection: AI_PREFERENCE_NSID,
           rkey: 'self',
+        },{
+          headers: {
+            "atproto-proxy": ""
+          }
         })
         return data.value as AIPreferenceRecord
       } catch (e) {
@@ -83,6 +87,10 @@ export function useUpdateAIPreferencesMutation({
         collection: AI_PREFERENCE_NSID,
         rkey: 'self',
         record: next,
+      },{
+        headers: {
+          "atproto-proxy": ""
+        }
       })
     },
     onMutate: async patch => {

